hs_core/templates/pages/genericresource.html

```diff
@@ -1494,10 +1494,10 @@ <h4 class="modal-title">Confirm files deletion</h4>
                 }
 
                 // If does not exist, add it
-                if (!exists && keyword[i] != "") {
+                if (!exists && keyword[i] != "" && keyword[i].length <= 100) {
```

Comment:
Thanks @zhiyuli for the quick review. If we see further issues with keywords update, I will make the delete and update operations part of a transaction.